src/sig-server/bin.js

```diff
@@ -3,13 +3,20 @@
 
 'use strict'
 
+// Usage: $0 [--host <host>] [--port <port>] [--disable-metrics]
+
 const signalling = require('./index')
-const argv = require('minimist')(process.argv.slice(2))
+const minimist = require('minimist')
+const argv = minimist(process.argv.slice(2), {
+  p: 'port',
+  h: 'host',
+  'disable-metrics': 'disableMetrics'
+})
```

Comment:
These need to be under the alias option to work
```suggestion
const argv = minimist(process.argv.slice(2), {
  alias: {
    p: 'port',
    h: 'host',
    'disable-metrics': 'disableMetrics'
  }
})
```